cli/transmission-cli.1

```diff
@@ -74,17 +74,41 @@ Enable portmapping via NAT-PMP or UPnP
 Disable portmapping
 .It Fl p -port Ar port
 Set the port to listen for incoming peers. (Default: 51413)
-.It Fl t -tos Ar tos
-Set the peer socket TOS for local router-based traffic shaping.
-Valid values are
+.It Fl -tos Ar tos
+.nf
+Use a ToS or DSCP name to set the peer socket service type for local router-based traffic shaping. (Default: le)
+Valid values are:
+.fi
 .Pp
+.Dq af11 ,
+.Dq af12 ,
+.Dq af13 ,
+.Dq af21 ,
+.Dq af22 ,
+.Dq af23 ,
+.Dq af31 ,
+.Dq af32 ,
+.Dq af33 ,
+.Dq af41 ,
+.Dq af42 ,
+.Dq af43 ,
+.Dq cs0 ,
+.Dq cs1 ,
+.Dq cs2 ,
+.Dq cs3 ,
+.Dq cs4 ,
+.Dq cs5 ,
+.Dq cs6 ,
+.Dq cs7 ,
+.Dq ef ,
+.Dq le ,
 .Dq default ,
 .Dq lowcost ,
 .Dq throughput ,
 .Dq reliability ,
 .Dq lowdelay ,
```

Comment:
```suggestion
.Dq lowdelay
```
Eliminates trailing `,` from manpage.

Reply:
fixed in 334ac3a0bbd21976a7578e1c03424d9f85b8acb0